language-design.md

```diff
@@ -102,7 +289,43 @@ you do so? If not, why not?
 
 **Response**
 
-
+It may be possible to have a language be in the same "grain" as a natural 
+language without actually having the associated ambiguity. If it has a 
+well defined structure and syntax, then the flexibility described in the
+excerpt above may not be an issue.
+
+In Pavlus, it is described that "finding natural-language replacements for 
+some of the more abstruse syntax went a long way" [Pavlus 2012]. At first
+glance, this seems to be a direct contradiction to the experiment described
+in the above quote from Cook. However, in the case of Pavlus, novices were
+the ones being experimented on. For us, the more natural language-like 
+syntax in Pavlus was less easy to understand than the Java syntax. In Cook, 
+programmers were the ones using the natural language rather than novices.
+It would be interesting to perform an experiment about whether novices
```

Comment:
I would also be interested to see what happens if these novices become professional programmers and never use anything but the natural language-like syntax. While the compact syntax is easier for us, that's just a matter of familiarity. If there was a paradigm shift for all new developers to use natural language-like GPLs, I think we'd see a gradual change over time.